src/renderers/webgpu/WebGPUBackend.js

```diff
@@ -279,6 +279,17 @@ class WebGPUBackend extends Backend {
 
 	}
 
+	/**
+	 * Returns `true` if the backend requires a framebuffer target with type `HalfFloat`.
+	 *
+	 * @return {Boolean} Whether the backend requires a framebuffer target with type `HalfFloat` or not.
+	 */
+	needsHalfFloatFrameBufferTarget() {
+
+		return ( this.parameters.outputType === HalfFloatType );
```

Comment:
I think the `needsHalfFloatFrameBufferTarget()` test should be valid for both backends, because it's about a target-texture and not the final output, so this could affect the difference in output from each backend.